3_ProtectionProfile/BiocPP.adoc

```diff
@@ -186,36 +193,50 @@ The evaluation of the above security functions is out of scope of this PP-Module
+For enhanced security that is easy to use, mobile device may implement biometric verification on a device once it has been “unlocked”. The initial unlock is generally done by a PIN/password which is required at startup (or possibly after some period of time), and after that the user is able to use an own biometric characteristic to unlock access to the mobile device. In this use case, the mobile device is not supposed to be used for security sensitive services through the biometric verification.
 
-Main concern of this use case is the accuracy of mobile biometric verification (i.e. FAR/FMR and FRR/FNMR) and basic level of presentation attacks. Security assurance for mobile device that the TOE relies on should be handled by the base PP.
+Main concern of this use case is the accuracy of the biometric verification (i.e. FAR/FMR and FRR/FNMR). Security assurance for mobile device that the TOE relies on should be handled by the base PP.
 
-This use case assumes that the mobile device is configured correctly to enable the mobile biometric verification by the biometric system administrator. The user of the mobile device can act as the biometric system administrator in this use case.
+This use case assumes that the mobile device is configured correctly to enable the biometric verification by the biometric system administrator. The user of the mobile device can act as the biometric system administrator in this use case.
 
-It is also assumed that the user enrols his/herself correctly, following the guidance provided by the TOE. Attacks during enrolment may be out of scope, but optionally addressed. FTE is not a security relevant criterion for this use case.
+It is also assumed that the user enrols his/herself correctly, following the guidance provided by the TOE. Presentation attacks during biometric enrolment and verification may be out of scope, but optionally addressed. FTE is not a security relevant criterion for this use case.
 
-==== USE CASE 2: Biometric verification for security sensitive service
+===== USE CASE 2: Biometric verification for security sensitive service
 
-This use case is an example of another use case that isn’t considered in this PP-Module. Another PP-Module should be developed at higher assurance level for this use case.
+This use case is an example of another use case that isn’t considered in this PP-Module. Another PP or PP-Module should be developed at higher assurance level for this use case.
 
 Mobile devices may be used for security sensitive services such as payment transactions and online banking. Verification may be done by the biometric for convenience instead of PIN/password to access such security sensitive services.
 
-The requirements for the TOE focus on the biometric performance (FTE, FAR/FMR and FRR/FNMR) and higher level of presentation attack.
+The requirements for the TOE focus on the biometric performance (FTE, FAR/FMR and FRR/FNMR) and presentation attack detection.
+
+== Consistency rationale
```

Comment:
I would expect this to be specified in the PPC since consistency would be part of the rationale of the PPC, not the PPM

Reply:
I added this consistency rationale because the CC part 1 requires it (See figure 11 in CC Part1). I personally think that the CC should be modified and this rationale should be described in the PPC but we should follow the current CC.

Reply:
I would think we could put in something here that would say that the consistency rationale is provided in an appropriate PPC, so we are referencing where to find it from here without needing to specifically place it or any references to the base-PP in this document. Maybe we also consider providing a reference to a web page (i.e. something on the github.io site) that lists the specific base-PPs the PPM has been approved to be used with so there is a clear reference to the other docs.